src/main/java/com/microsoft/azure/util/AzureMsiCredentials.java

```diff
@@ -32,5 +95,14 @@ public String getDisplayName() {
         public int getDefaultMsiPort() {
             return DEFAULT_MSI_PORT;
         }
+
+        public ListBoxModel doFillAzureEnvNameItems() {
+            ListBoxModel model = new ListBoxModel();
+            model.add(AzureCredentials.Constants.ENV_AZURE);
+            model.add(AzureCredentials.Constants.ENV_AZURE_CHINA);
+            model.add(AzureCredentials.Constants.ENV_AZURE_GERMANY);
+            model.add(AzureCredentials.Constants.ENV_AZURE_US_GOVERNMENT);

```

Comment:
It may be better to move the `Constants` class to a top level class as it's now shared by two distinct classes.